### ISSUE TITLE
query-exporter binaries

### ISSUE THREAD

**Issue by tmihaldinec** [albertodonato/query-exporter#15]
Hi,

is it possible to get query-exporter binaries without snap.. 
problem is that snap cannot be installed on system which is highly secured and without internet access

Many thanks
tomislav

**Comment by albertodonato** [comment]
What do you mean by binaries?
You can install it along with its dependencies in a virtualenv, which can be copied around easily.

Packaging for distributions would be a lot of work, as it would also need to include dependencies where not available in the distro.

**Comment by tmihaldinec** [comment]
Hi,
what do you mean by installing it in virtual environment? like docker?
what i wanted to do is to install snap somewhere else and copy only what is needed to other servers

problem is that query-exporter is somehow alias of snap

[root@lvpgrafanasrv01 current]# ls -al /var/lib/snapd/snap/bin/query-exporter
lrwxrwxrwx 1 root root 13 Nov  6 12:23 /var/lib/snapd/snap/bin/query-exporter -> /usr/bin/snap

basically what i want to do is to copy query-exporter binary and all dependancies to other server and run it there

how can i do it?
thanks
t

**Comment by albertodonato** [comment]
I meant a python virtualenv, created with https://virtualenv.pypa.io/en/stable/

You can create a relocatable virtualenv in a dir, then `pip install query-exporter` (and the dependencies for database backends you need) in there.

That directory can be copied to a server, and you can run the `bin/query-exporter` script from that dir.
